homeassistant/components/unifiprotect/__init__.py

```diff
@@ -41,6 +44,58 @@
 SCAN_INTERVAL = timedelta(seconds=DEFAULT_SCAN_INTERVAL)
 
 
+async def _async_migrate_data(
+    hass: HomeAssistant, entry: ConfigEntry, protect: ProtectApiClient
+) -> None:
+
+    registry = er.async_get(hass)
+    to_migrate = []
+    for entity in er.async_entries_for_config_entry(registry, entry.entry_id):
+        if entity.platform == Platform.BUTTON and "_" not in entity.unique_id:
+            to_migrate.append(entity)
+
+    if len(to_migrate) == 0:
+        return
+
+    _LOGGER.info("Migrating %s reboot button entities ", len(to_migrate))
+    bootstrap = await protect.get_bootstrap()
+    count = 0
+    for button in to_migrate:
+        device = None
+        for model in DEVICES_THAT_ADOPT:
+            attr = f"{model.value}s"
+            device = getattr(bootstrap, attr).get(button.unique_id)
+            if device is not None:
+                break
+
+        if device is not None:
```

Comment:
```suggestion
        if device is None:
            continue
            
```

You can save a bit of indent if you reverse the condition and continue